Avoid a nearly singular matrix in lapack testing, which can trigger an error depending
on FMA. Swapping matrix dimensions is sufficient here since the matrices are generated
using a random number generator, and we were just unlucky here.

Author: Bart Oldeman <bart.oldeman@calculquebec.ca>
--- OpenBLAS-0.3.23/lapack-netlib/TESTING/dgd.in.orig	2023-06-05 12:12:08.222632285 +0000
+++ OpenBLAS-0.3.23/lapack-netlib/TESTING/dgd.in	2023-06-05 12:12:19.323715597 +0000
@@ -1,6 +1,6 @@
 DGS               Data for the Real Nonsymmetric Schur Form Driver
 5                 Number of matrix dimensions
-2 6 10 12 20 30   Matrix dimensions
+6 2 10 12 20 30   Matrix dimensions
 1 1 1 2 1         Parameters NB, NBMIN, NXOVER, NS, NBCOL
 10                Threshold for test ratios
 .TRUE.            Put T to test the error exits
--- OpenBLAS-0.3.23/lapack-netlib/TESTING/sgd.in.orig   2023-06-06 11:01:50.512947527 +0000
+++ OpenBLAS-0.3.23/lapack-netlib/TESTING/sgd.in        2023-06-06 11:02:05.318078733 +0000
@@ -1,6 +1,6 @@
 SGS               Data for the Real Nonsymmetric Schur Form Driver
 5                 Number of matrix dimensions
-2 6 10 12 20 30   Matrix dimensions
+6 2 10 12 20 30   Matrix dimensions
 1 1 1 2 1         Parameters NB, NBMIN, NXOVER, NS, NBCOL
 10                Threshold for test ratios
 .TRUE.            Put T to test the error exits
